blob/fileblob/example_test.go

```diff
@@ -65,9 +66,16 @@ func Example_openBucket() {
 	}
 	defer os.RemoveAll(dir)
 
+	// On Unix, append the dir to "file://".
+	// On Windows, convert "\" to "/" and add a leading "/":
+	dirpath := filepath.ToSlash(dir)
```

Comment:
Optional: Seeing this in the example directly, it kinda feels like we should have a utility function that does this, although I realize the user would probably just call the `fileblob.OpenBucket` call directly. WDYT?

Reply:
I think the user is more likely to be constructing the URLs by hand than to be writing code to generate them. I agree with you that if they have the directory in code, they should just call `OpenBucket` directly, and not go through the URL path.